test/12_nullablearray.jl

```diff
@@ -965,4 +965,66 @@ for isordered in (false, true)
     end
 end
 
+
+# Tests of vcat of NullableCategoricalArray
+
+for (CA,A) in ((:NullableCategoricalArray,:NullableArray),)
```

Comment:
I only had one for-loop at first but wanted to hear your comments on the approach. The test code within the body of the for-loop is identical in the two files.